torch/quantization/fx/quantize.py

```diff
@@ -168,23 +192,74 @@ def __init__(self):
     def _qat_swap_modules(self, root):
         convert(root, mapping=get_qat_module_mappings(), inplace=True, remove_qconfig=False)
 
-    def _generate_qconfig_map(self, root, input_graph):
-        def get_qconfig(module):
-            return module.qconfig if hasattr(module, 'qconfig') else None
+    def _generate_qconfig_map(self,
+                              root,
+                              input_graph,
+                              flattened_qconfig_dict,
+                              module_name_regex_qconfig_list):
```

Comment:
any chance we can annotate these types in the docblock, to help with readability?

Reply:
changed to using OrderedDict, hopefully it's clearer now